WordPress/Sniffs/NamingConventions/PrefixAllGlobalsSniff.php

```diff
@@ -21,7 +21,8 @@
  * @since   0.12.0
  * @since   0.13.0 Class name changed: this class is now namespaced.
  * @since   1.2.0  Now also checks whether namespaces are prefixed.
- * @since   2.2.0  Now also checks variables assigned via the list() construct.
+ * @since   2.2.0  - Now also checks variables assigned via the list() construct.
+ *                 - Will now ignore globals functions which are marked as @deprecated.
```

Comment:
This sounds a bit odd. Maybe it would be better to say: `Ignores global functions marked as @deprecated`?

Reply:
To me, the difference between our phrasing is "what has changed" and "what it does", with the first being what is intended as it is a changelog, not part of the generic class description.

Happy to change it, but in that case, I'd like to see a suggestion which reads like a changelog still.

Reply:
Yeah, I guess I had class description in mind vs changelog which this is. When I look at it that way the `Sniff will now ignore global functions which are marked as @deprecated` sounds better.

In that case, I think that just the `globals functions` should be changed with `global functions` no?

Reply:
Thanks for catching that. I've fixed that and made a minor tweak to the text. Hope it's ok now.